packages/wallets/src/components/DerivAppsSection/DerivAppsGetAccount.tsx

```diff
@@ -1,13 +1,86 @@
-import React from 'react';
+import React, { useCallback, useEffect, useMemo } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useActiveWalletAccount, useCreateNewRealAccount, useSettings } from '@deriv/api';
+import { toMoment } from '../../../../shared/src/utils/date';
+import { Success } from '../../features/cfd/screens/Success';
+import useDevice from '../../hooks/useDevice';
 import DerivApps from '../../public/images/deriv-apps.svg';
 import { THooks } from '../../types';
-import { WalletButton, WalletText } from '../Base';
+import { ModalStepWrapper, WalletButton, WalletText } from '../Base';
+import { useModal } from '../ModalProvider';
 
 type TProps = {
     isMaltaWallet?: THooks.ActiveWalletAccount['is_malta_wallet'];
 };
 
 const DerivAppsGetAccount: React.FC<TProps> = ({ isMaltaWallet }) => {
+    const { hide, show } = useModal();
+    const { isDesktop } = useDevice();
+    const { data: activeWallet } = useActiveWalletAccount();
+    const { isSuccess: isAccountCreationSuccess, mutate: createNewRealAccount } = useCreateNewRealAccount();
+    const {
+        data: { country_code: countryCode, date_of_birth: dateOfBirth, first_name: firstName, last_name: lastName },
+    } = useSettings();
+    const history = useHistory();
+
+    const landingCompanyName = activeWallet?.landing_company_name?.toLocaleUpperCase();
+
+    const footer = useMemo(
```

Comment:
could it be a separate? would also naturally create a space for styles for it,

Reply:
or if you do not want separate component - then might be good to give it a class derived from "DerivAppsGetAccount" and create stylesheet for DerivAppsGetAccount and put its styles there (as this function effectvely belongs to DerivAppsGetAccount at the moment).

Reply:
Extracted out to different component.